react/features/subtitles/components/LanguageSelectorDialog.web.tsx

```diff
@@ -6,35 +6,37 @@ import { IState } from '../../app/types';
 // @ts-ignore
 import { Dialog } from '../../base/dialog';
 // @ts-ignore
-import { LANGUAGES, TRANSLATION_LANGUAGES_HEAD, TRANSLATION_LANGUAGES_EXCLUDE } from '../../base/i18n';
+import { TRANSLATION_LANGUAGES, TRANSLATION_LANGUAGES_HEAD } from '../../base/i18n';
 import { connect } from '../../base/redux/functions';
 // @ts-ignore
-import { updateTranslationLanguage, setRequestingSubtitles, toggleLangugeSelectorDialog } from '../actions';
+import { updateTranslationLanguage, setRequestingSubtitles, toggleLanguageSelectorDialog } from '../actions';
 
 import LanguageList from './LanguageList';
 
 interface ILanguageSelectorDialogProps {
     _language: string;
     t: Function;
+    translationLanguages: Array<string>;
+    translationLanguagesHead: Array<string>;
 }
 
 /**
  * Component that renders the subtitle language selector dialog.
  *
  * @returns {React$Element<any>}
  */
-const LanguageSelectorDialog = ({ _language }: ILanguageSelectorDialogProps) => {
+const LanguageSelectorDialog = ({ _language, translationLanguages, translationLanguagesHead }:
+                                    ILanguageSelectorDialogProps) => {
+
     const dispatch = useDispatch();
     const off = 'transcribing.subtitlesOff';
     const [ language, setLanguage ] = useState(off);
 
-    const importantLanguages = TRANSLATION_LANGUAGES_HEAD.map((lang: string) => `languages:${lang}`);
-    const fixedItems = [ off, ...importantLanguages ];
-
-    const languages = LANGUAGES
-        .filter((lang: string) => !TRANSLATION_LANGUAGES_EXCLUDE.includes(lang))
-        .map((lang: string) => `languages:${lang}`)
-        .filter((lang: string) => !(lang === language || importantLanguages.includes(lang)));
+    const languagesHead = translationLanguagesHead.map((lang: string) => `translation-languages:${lang}`);
+    const fixedItems = [ off, ...languagesHead ];
```

Comment:
What does "fixed" mean here? I feel like a comment explaining what you're doing here would help.